src/css/index.scss

```diff
@@ -1,2 +1,356 @@
-@import './global';
-@import './information';
+$primary_color: #ffffff;
+$secondary_color: #070a0e;
+$tertiary_color: #94a3ba;
+$btn_hover_color: #ff7675;
+
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+}
+
+.bgimg {
+  background: url('../css/cv.jpg') fixed no-repeat;
+  background-size: cover;
+  background-position: center;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  filter: blur(5px);
+}
+
+.header {
+  width: 100%;
+  padding: 20px 0 0 0;
+  height: 100px;
+  position: fixed;
+  z-index: 1;
```

Comment:
Might consider increate the z-index since the header should be on top of basically everything.

You will be running into problems when element below is absolute positioned with a higher zIndex.